examples/13_regions.cpp

```diff
@@ -0,0 +1,261 @@
+#include <openPMD/regions/Regions.hpp>
+
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <random>
+#include <vector>
+
+////////////////////////////////////////////////////////////////////////////////
+
+void point_example() {
+  using namespace openPMD::Regions;
+  using namespace std;
+
+  cout << "\n"
+       << "Points (with dimension known at compile time):\n";
+
+  cout << "  Define two points:\n";
+  Point<int, 2> x{1, 2}, y{4, 5};
+  cout << "    x: " << x << "\n"
+       << "    y: " << y << "\n";
+
+  cout << "  Arithmetic operations:\n";
+  auto z = x + 2 * y;
+  cout << "    z: " << z << "\n";
+
+  cout << "  Unit vectors pointing in direction d:\n";
+  auto u0 = Point<int, 2>::unit(0);
+  auto u1 = Point<int, 2>::unit(1);
+  cout << "    u0: " << u0 << "\n"
+       << "    u1: " << u1 << "\n";
+  cout << "  A points with all elements the same:\n";
+  auto p3 = Point<int, 2>::pure(3);
+  cout << "    p3: " << p3 << "\n";
+
+  cout << "  Element-wise operations:\n";
+  using std::abs, std::max;
```

Comment:
isn't `using namespace std;` already in scope?

Reply:
Yes it is.